README.md

```diff
@@ -1,31 +1,56 @@
 # setup-cli
 
-This is a composite GitHub Action that sets up the Databricks CLI (preview version `>=0.100`) in your GitHub Actions workflow.
+Easily install the [Databricks CLI](https://github.com/databricks/cli) in your environment.
```

Comment:
How about

```suggestion
setup-cli is a script to easily install the [Databricks CLI](https://github.com/databricks/cli) in your environment.
```

Reply:
It's not a script though.

Reply:
What would you call it?

Reply:
I can't think of one that doesn't sound contrived. Do you have alternatives?

Reply:
How about "setup-cli makes it easy to install .... It provides a ~composite~ GitHub Action and a portable script that can be used in most CI/CD systems and development environments."

Reply:
Thanks, applied.